ihmc-footstep-planning/src/main/resources/us/ihmc/footstepPlanning/SnappingTerrainExtractor.cu

```diff
@@ -184,7 +184,8 @@ __global__ void computeTerrainData(float *heightMap, size_t pitchHeightMap,
     }
 
     // This is the actual height of the snapped foot
-    float snap_height = z / n;
+    float *heightValue = (float *) ((char *)heightMap + y_index * pitchHeightMap) + x_index;
+    float snap_height = *heightValue;
 
```

Comment:
Why this change? `z/n` is the height from the best-fit plane and probably removes some noise